codalab/worker/worker_run_state.py

```diff
@@ -125,6 +125,22 @@ class RunStage(object):
 DependencyToMount = namedtuple('DependencyToMount', 'docker_path, child_path, parent_path')
 
 
+def log_bundle_transition(
+    bundle_uuid: str,
+    previous_state: str,
+    next_state: str,
+    reason: str = '',
+    level: int = logging.INFO,
+    warn: bool = False,
```

Comment:
Why do we still need `warn` if we now have `level`? We should show the stacktrace if it's an error.

Reply:
Is it helpful / necessary to show the stacktrace if it's a warning?

Reply:
I'm not sure why the boolean is needed when `level` has the same information.

Reply:
Usually stacktraces are shown with errors and not warnings.

Reply:
oh I thought if the level is warning, probably stacktrace is also useful so I changed the code accordingly. You can check the most up to date code.